.github/workflows/test_objectivec.yml

```diff
@@ -68,17 +68,27 @@ jobs:
     strategy:
       fail-fast: false   # Don't cancel all jobs if one fails.
       matrix:
-        PLATFORM: ["ios", "macos", "tvos", "watchos"]
+        PLATFORM: ["ios", "macos", "tvos", "watchos", "visionos"]
         CONFIGURATION: ["Debug", "Release"]
-    name: CocoaPods ${{ matrix.PLATFORM}} ${{ matrix.CONFIGURATION}}
-    runs-on: macos-12
+        include:
+          - OS: macos-12
+          - XCODE: Xcode_14.1
+          - PLATFORM: "visionos"
+            OS: macos-14
```

Comment:
https://github.com/actions/runner-images seems to say this image is _beta_, any idea the stability?  We won't want something on CI that can't be relied on.

Reply:
Looks like https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#xcode might have Xcode 15.2, so maybe only macos-13 is needed.

But now that I think about it, doesn't a visionOS build really need to be arm?  Does Apple fully support the SDK on an Intel based machine?

Reply:
When I run pod lib lint based on macos-13 image and Xcode 15.2，the error's description as `[visionOS] unknown: Encountered an unknown error (Could not find a `xros` simulator (valid values: ios, tvos, watchos).` appeared also.
See: https://github.com/growingio/growingio-sdk-ios-utilities/actions/runs/7984766703/job/21802111821
And: https://github.com/actions/runner-images/issues/8144#issuecomment-1810765004

Reply:
In my latest workflow file, [there](https://github.com/growingio/growingio-sdk-ios-utilities/blob/4d3c0eb227f6b11b944836da64ef9fced89ae4e6/.github/workflows/pod_lint.yml#L15) is `macos-14`，but `macos-14-arm64` [get to used](https://github.com/growingio/growingio-sdk-ios-utilities/actions/runs/7985131966/job/21846237774#step:1:8) actually.